training-react/src/app/components/User/index.js

```diff
@@ -1,17 +1,17 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import userActions from '../../../redux/user/actions';
 import LocalStoreService from '../../../services/LocalStoreService';
-import Loading from '../Loading';
 import UserInfo from '../UserInfo';
 import UserEdit from '../UserEdit';
+import withLoadingScreen from '../LoadingScreen';
 
-import styles from './styles.scss';
+import UserDetails from './layout';
 
 const USER_SESSION = 'USER_SESSION';
+const UserDetailsWithLoading = withLoadingScreen(UserDetails);
```

Comment:
withLloading should be placed where you export the UserDetails component
```js
export default withLoadingScreen(UserDetails);
```
Then it's transparent when importing it.